views/extras.tmpl

```diff
@@ -94,429 +87,14 @@
-                // This probability is using the network parameter rather than the calculated value. So far, the ticket pool size seems stable enough to do that.
-                var pctChance = Math.pow(1 - 1/DCRThings.ticketPoolSize, blocksLeft)*100;
-                $("#ticketExpiryChance").children("span").html(pctChance.toFixed(2)+"% chance of expiry");
-            }
-            else{
-                var typeStub = txType == "Ticket" ? "eligible to vote" : "spendable";
-                ticketProgress.children("span").html("Immature, "+typeStub+" in "+blocksLeft+" blocks ("+(remainingTime/3600.).toFixed(1)+" hours remaining)");
-            }
-            ticketProgress.attr("aria-valuenow", confirmations).css({"width":(ratio*100).toString()+"%"});
-        }
-    }
-
-    // Check the block for mempool transactions on the address page.
-    function confirmAddrMempool(block){
-      var mempoolRows = $("[data-addr-tx-pending]");
-      mempoolRows.each(function(idx, tr){
-        var row = $(tr)
-        var txid = row.data('addr-tx-pending')
-        if(txInBlock(txid, block)){
-          var confirms = row.children('.addr-tx-confirms');
-          confirms.attr('data-tx-block-height', block.height);
-          row.removeAttr('data-addr-tx-pending');
-          row.children('.addr-tx-time').html(formatTxDate(block.time, false));
-          row.children('.addr-tx-age').children('span').attr("data-age", block.time).html(humanize.timeSince(block.time));
-          confirms.html("1")
-        }
-      });
-      var confirmTds = $('[data-tx-block-height]');
-      confirmTds.each(function(idx, td){
-        confirms = $(td);
-        confirms.html(block.height - parseInt(confirms.data('tx-block-height')) + 1);
-      });
-      var pendingCounters = $('[data-tx-confirmations-pending]');
-      pendingCounters.each(function(i,c){
-        var counter = $(c);
-        var txid = counter.data('tx-confirmations-pending')
-        if(txInBlock(txid, block)){
-          counter.removeAttr('data-tx-confirmations-pending')
-          counter.attr('data-confirmation-block-height', block.height).html('(1 confirmation)')
-        }
-      });
-    }
-
-</script>
-
-<script>
-    if (!ws) {
-        createWebSocket(window.location)
-    }
-</script>
-
-<script src="/js/notify.min.js"></script>
 {{end}}
 
 {{define "footer"}}
 <footer class="navbar-fixed-bottom">
     <div class="container d-flex justify-content-between align-items-center">
         <ul class="nav justify-contents-left nowrap blockcounter">
           <li class="nav-item text-left">
-            <span data-controller="main" data-target="main.age" data-main-lastblocktime="{{$.Tip.Time}}"></span> <span class="block-msg">since last block</span>
+            <span data-controller="time" data-target="time.age" data-main-lastblocktime="{{$.Tip.Time}}"></span> <span class="block-msg">since last block</span>
```

Comment:
The `data-main-lastblocktime` here is actually part of the stimulus naming convention, so it too should be changed to `data-time-lastblocktime`, so that stimulus can use the `data.has` function in `time_controller.js`. Appropriate updates are needed in other places where this attribute is referenced as well.